src/pages/Admin.jsx

```diff
@@ -65,6 +70,8 @@ const Admin = () => {
         Patients Management
       </Button>
       <Button
+        component={Link}
+        to="/In-Service"
```

Comment:
technically , if you peep into the App.jsx "/In-Service" does not exist.
what is in the App.jsx is    "/in-service"
observe that all other routes are all lowercases. Just consistence

Reply:
Change made